src/main/java/org/spongepowered/common/config/category/TileEntityActivationModCategory.java

```diff
@@ -33,15 +33,16 @@
 @ConfigSerializable
 public class TileEntityActivationModCategory extends ConfigCategory {
 
-    @Setting(value = "enabled", comment = "If 'false', tileentity activation rules for this mod will be ignored and always tick.")
+    @Setting(value = "enabled",
+            comment = "If 'false', tileentity activation rules for this mod will be ignored and always tick.")
     private boolean isEnabled = true;
-    @Setting(value = "default-block-range")
+    @Setting(value = "default-block-range", comment = "???")
```

Comment:
my config file has this for it, not sure where that comment went? `Default activation block range used for all tileentities unless overridden.`